packages/react-composites/tests/browser/call/app/index.tsx

```diff
@@ -43,13 +39,22 @@ function App(): JSX.Element {
 
   useEffect(() => {
     const initialize = async (): Promise<void> => {
-      const newAdapter = await createAzureCommunicationCallAdapter({
-        userId: fromFlatCommunicationIdentifier(userId) as CommunicationUserIdentifier,
-        displayName,
-        credential: new AzureCommunicationTokenCredential(token),
-        locator: { groupId: groupId }
-      });
-      setCallAdapter(wrapAdapterForTests(newAdapter));
+      if (params.useMockAdapter) {
+        setCallAdapter(new MockCallAdapter(state));
+      } else {
+        const displayName = verifyParamExists(params.displayName, 'displayName');
```

Comment:
At this point, I'd recommend creating a helper function for line 45 - 54. The code block here makes the conditional code flow harder to follow.